frontend/src/components/mainPage/scene/subcomponents/NavigationPanel.vue

```diff
@@ -30,7 +31,19 @@
                             this.setFrameId(this.currentFrame.id - 1);
                         },
                     },
-                    { icon: "mdi-play", action: () => {} },
+                    {
+                        icon: "mdi-play",
+                        action: () => {
+                            if (this.animationInterval) {
+                                clearInterval(this.animationInterval);
+                                this.animationInterval = null;
+                                this.icons[2].icon = "mdi-play";
```

Comment:
W "computed" dodałbym jakąś funkcję np. playStopIcon, która zwraca ikonkę, którą chcemy tutaj zmienić. W przypadku, gdyby np zmieniła by się kolejność ikonek i ta ikonka by miała index 3 a nie 2, to mamy jedno miejsce gdzie trzeba to zmienić. No i jak w kodzie ktoś zobaczy `this.playStopIcon` to to więcej mówi niż `this.icons[2]` (ktoś by musiał sprawdzić jaka ikonka jest pod 2 indeksem)